src/box/tuple.h

```diff
@@ -146,6 +146,13 @@ box_tuple_unref(box_tuple_t *tuple);
 API_EXPORT uint32_t
 box_tuple_field_count(box_tuple_t *tuple);
 
+/**
+ * Return the pointer to internal tuple data (pointer to the MsgPack Array).
+ * \param tuple tuple.
+ */
```

Comment:
I wouldn't call the tuple data as internal. At least, we have no internal/external tuple data representation to highlight it here. And we feed the same tuple data via iproto or when `msgpack.encode(<tuple>)` is used. So, this data format is public.

Reply:
The same about the changelog entry and the documentation request.